pulse/views/community_views.py

```diff
@@ -33,6 +36,56 @@ def createCommunity(request: HttpRequest) -> JsonResponse:
         )
     return JsonResponse(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+@api_view(["POST"])
+def approveCommunityRequest(request: HttpRequest, community_id: str) -> JsonResponse:
+    """
+    Approve a community request by setting approved=True. 
+    Adds the owner as a community member if approval is successful.
+
+    Args:
+        request (HttpRequest): The incoming HTTP request.
+        community_id (str): The ID of the community to approve.
+
+    Returns:
+        JsonResponse: A success message or an error message.
+    """
+    # Get the community or return 404 if not found
```

Comment:
It is probably a good idea to still double-check here that the user who is approving this has admin privileges (otherwise non-admin can reach this endpoint by entering the corresponding URL) -> basically just never _only_ trust the frontend lol